src/playground-project.ts

```diff
@@ -712,6 +712,59 @@ const typeEnumToMimeType = (type?: string) => {
       return 'text/html; charset=utf-8';
     case 'css':
       return 'text/css; charset=utf-8';
+    // taken from MDN's common MIME types
+    // https://developer.mozilla.org/en-US/docs/Web/HTTP/Basics_of_HTTP/MIME_types/Common_types
```

Comment:
In the interest of brevity, I think I'd skip these:

```
bmp
gif
tif
tiff
avi
mp4
mpeg
ogv
3gp
3g2
aac
mid
midi
oga
opus
wav
```

I think most of these media formats are already pretty edge case anyway, Let's just include the "modern" and old-but-too-common-on-the-web-to-ignore formats. Users can always explicitly set the mime type with the `contentType` field in the JSON config.